pages/feed/index.tsx

```diff
@@ -41,20 +43,26 @@ const Feeds = () => {
 
   useEffect(() => {
     if (selectedRepo) {
-      router.push(`/feed?repo=${selectedRepo}`);
+      router.push(`/feed?repo=${selectedRepo}`).catch(console.error);
```

Comment:
IMHO this is not caught because it is not awaited

Reply:
Yeah :sweat_smile: 
> calling router.push() multiple times within the same Next.js page in different places/throughout different re-renders seems to cause this error in some cases
source: https://stackoverflow.com/a/73344411/15301961